.travis.yml

```diff
@@ -1,38 +1,32 @@
-language: node_js
-dist: trusty
-addons:
-  apt:
-    packages:
-      # This is required to run new chrome on old trusty
-      - libnss3
-node_js:
- - 'node'
- - 'lts/*'
-sudo: false
-
-cache: 
-  directories:
-    # cache the larger node_modules we use throughout the tree 
-    # - "node_modules"

```

Comment:
We are no longer caching node_modules and other downloadable folders?

Reply:
Apparently you're supposed to cache `~/.npm` instead of `node_modules`, per the maintainer of npm.  This will still mean we cache node modules, but at a higher level (at the npm cache level) vs in our repo dir.  It will end up having the same effect.

For the release VM bits, I'd like to download from gh-pages instead of caching, now that we have them in a release.